packages/core/src/shared/constants.ts

```diff
@@ -189,6 +189,14 @@ export const amazonQHelpUrl = 'https://aws.amazon.com/q/'
 export const amazonQVscodeMarketplace =
     'https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode'
 
+// URLs for Amazon EventBridge Scheduler
```

Comment:
The addition of EventBridge Scheduler documentation URLs is good. However, consider the following improvements:

1. Use a more specific naming convention for the constants, such as prefixing them with `EVENTBRIDGE_SCHEDULER_` to avoid potential naming conflicts.

2. Consider grouping related constants together, possibly in a namespace or object, to improve organization as the number of constants grows.

3. Add comments explaining what each URL points to, to improve maintainability.
```suggestion
// URLs for Amazon EventBridge Scheduler
export const EVENTBRIDGE_SCHEDULER_URLS = {
    /** Documentation for creating schedules */
    CREATE_SCHEDULE: '
    /** Documentation for managing schedules via API Gateway */
    API_GATEWAY: '
    /** Documentation for setting up IAM role permissions */
    ROLE_PERMISSIONS: '
};
```